azure/v12.0.1/README.md

```diff
@@ -0,0 +1,20 @@
+## :zap:  Giant Swarm Release 12.0.1 for Azure :zap:
+
+**If you are upgrading from 12.0.0, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release upgrades external-dns app to v0.7.3.
+
+**Important Warning**
+During master upgrade from 11.4.0 to 12.0.x, within the time frame of 30 seconds we had noticed a spike in requests failures. This is most likely caused by Azure CNI upgrade and despite our great efforts, we had not found a solution to maintain upgrade path and avoid this disturbance. Please keep this in mind when scheduling an upgrade window, and contact your SE if you have further questions.
+
+Note that as of 12.0.0 release **NGINX Ingress Controller App** is **optional and not pre-installed** component on Azure. You can find more info in [12.0.0 release notes](https://github.com/giantswarm/releases/tree/master/azure/v12.0.0).
```

Comment:
@sslavic do we need this here? It seems out of place with the rest of the content for this release. Also, we mention it in the optional apps announcements.

Reply:
Removed. PTAL